src/transformers/feature_extraction_sequence_utils.py

```diff
@@ -107,6 +108,8 @@ def pad(
                   different lengths).
             max_length (:obj:`int`, `optional`):
                 Maximum length of the returned list and optionally padding length (see above).
```

Comment:
This should be adapted to also mention truncation now:
```suggestion
                Maximum length of the returned list and optionally padding or truncation length.
```